### PR TITLE
Adding sounds to stamps

### PR DESCRIPTION
**Pull request by Lomcastar**
## About the PR
Now it is more pleasant to put stamps.
Almost all stamps have the sound of a hand stamp.
But the DENIED and APPROVED stamps have an automatic stamp sound, which in my opinion sounds more pleasant.

**Media**

https://github.com/space-wizards/space-station-14/assets/129697969/c4fd8ec8-0b9b-485b-bc0b-e05ce7a204be

- [x] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**
:cl:
- add: Added stamp sounds.

### CODE REVIEW

File: Content.Shared/Paper/StampComponent.cs [hunk 1]
```diff
@@ -13,5 +13,8 @@ public sealed class StampComponent : Component
         /// </summary>
         [DataField("stampState")]
         public string StampState { get; set; } = "paper_stamp-generic";
+
+        [DataField("sound")]
+        public string Sound = string.Empty;
```
Comment (mirrorcult) [inline Content.Shared/Paper/StampComponent.cs:18]: use a `SoundSpecifier`, not a string path

**Review by mirrorcult — CHANGES_REQUESTED** [review]

File: Resources/Audio/Items/Stamp/thick_stamp_nosub.ogg
Comment (mirrorcult) [inline Resources/Audio/Items/Stamp/thick_stamp_nosub.ogg:1]: unused sound? whats the diff between sub vs nosub

Comment (Lomcastar) [reply Resources/Audio/Items/Stamp/thick_stamp_nosub.ogg:1]: very little difference in sound

Comment (Lomcastar) [reply Resources/Audio/Items/Stamp/thick_stamp_nosub.ogg:1]: is it better to remove it if it is not used?

Comment (mirrorcult) [reply Resources/Audio/Items/Stamp/thick_stamp_nosub.ogg:1]: no reason to keep it if its basically the same sound

**Comment by Lomcastar** [comment]
hmmmmmmmm... ok...

**Comment by CaptainSqrBeard** [comment]
Make clown stamp honk and mime stamp silent 😄

**Comment by Lomcastar** [comment]
@CaptainSqrBeard The idea is good but.
The seal of the mime is the base for all other seals.
A clown needs a special sound. Something between a stamp and a HONK.

**Comment by Emisse** [comment]
thunk

**Pull request merged**